lms/djangoapps/courseware/tests/test_split_module.py

```diff
@@ -123,7 +123,7 @@ def _check_split_test(self, user_tag):
         content = resp.content
 
         # Assert we see the proper icon in the top display
-        self.assertIn('<a class="{} inactive progress-0 nav-item"'.format(self.ICON_CLASSES[user_tag]), content)
+        self.assertIn('<button class="{} inactive progress-0 nav-item"'.format(self.ICON_CLASSES[user_tag]), content)
```

Comment:
Nit: testing against rendered HTML is fragile.  Class ordering, whitespace, and quote character choice are all semantically meaningless, but constrained by the test.  Better would be something like (pseudocode that uses non-existent methods):

    button = self.get_button(content, class='progress-0')
    self.assertIn(self.ICON_CLASSES[user_tag], button.attrib['class'])